test/performance/test_backscatter.py

```diff
@@ -0,0 +1,17 @@
+from backscatter import dmap
```

Comment:
Is this a useful file to include for testing in the future?

Reply:
It was a comparison against yours for time performance, also a reminder of how to use your code. If you don't see a need for this test file then I can remove it.

Reply:
Yeah I would say this can go. If its super critical later on we can always look back at the commits.